sale_generator/models/sale_generator.py

```diff
@@ -0,0 +1,116 @@
+#  @author Sébastien BEAU <sebastien.beau@akretion.com>
+#  @author Abdessamad HILALI <abdessamad.hilali@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class SaleGenerator(models.Model):
+    _name = "sale.generator"
+    _description = "Sale order generator"
+
+    name = fields.Char(string="Generator", default="/")
+    partner_ids = fields.Many2many(
+        comodel_name="res.partner", string="Partner"
+    )
+    sale_ids = fields.One2many(
+        comodel_name="sale.order", inverse_name="generator_id", string="Sales"
+    )
+    tmpl_sale_id = fields.Many2one(
+        comodel_name="sale.order", string="Sale Template", required=True,
+    )
+    date_order = fields.Datetime(
+        string="Date", oldname="date", default=fields.Datetime.now()
```

Comment:
oldname no more used
```suggestion
        string="Date", default=fields.Datetime.now()
```

Reply:
thanks David, I must've done some bad force push operations